tests/test_create_consensus_network.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/env/bin python3
+
+import unittest
+import filecmp
+import sys
+import os
+import shutil
+from SJARACNe.bin.create_consensus_network import create_consensus_network as cn
+from SJARACNe.bin.create_consensus_network import create_enhanced_consensus_network as ecn
+from SJARACNe.bin.create_consensus_network import uprob
+
+class TestConsensusNetwork(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.path = './test_answers'
+        if self.path is None:
+            os.mkdir('./test_answers')
+        cn('./tests/inputs/adjmat_dir', 0.05, './test_answers')
+
+    
+    def test_consensus_network_3col(self):
+        self.assertTrue(filecmp.cmp('./tests/answerkey/consensus_network_3col_.txt', './test_answers/consensus_network_3col_.txt'))
+        
+    def test_bootstrap_info(self):
+        self.assertTrue(filecmp.cmp('./tests/answerkey/bootstrap_info_.txt', './test_answers/bootstrap_info_.txt'))
+
+    def test_parameter_info(self):  
+        #delete last line of parameter info, as it will not match with answer key
+        with open('./test_answers/parameter_info_.txt', 'r') as info:
+            lines = info.readlines()
+            lines = lines[:-1]
+            info.close()
+        with open('./test_answers/parameter_info_.txt', 'w') as info:
+            info.writelines(lines)
+            info.close()
+        self.assertTrue(filecmp.cmp('./tests/answerkey/parameter_info_.txt', './test_answers/parameter_info_.txt'))
+
+
+    '''
+    def test_enhanced_consensus_network(self):
```

Comment:
Why is it commented out?

Reply:
Because this test takes 10-ish minutes to run using the network from BRCA100.exp. The test works, but I think it would be ideal to have a smaller network to create the enhanced one for testing.